distributions/demo-google-deployment/api/build.gradle

```diff
@@ -95,7 +115,7 @@ task createApiServerDockerfile(type: Dockerfile) {
     copyFile("build/libs/api-all.jar", "/app/api.jar")
     // TODO: make debug flag driven
     defaultCommand("java",
-            "-Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005",
+            "-Xrunjdwp:transport=dt_socket,server=y,suspend=n",
```

Comment:
Looks like the adress got dropped from this, is that intentional?

Reply:
Or wait, is this for the production env? Do we need this flag at all then?

Reply:
Exactly - this is the production image so I didn't want to expose debugging. I'm removing the TODO here (and leaving it for the local task)